tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -75,13 +92,23 @@ Amend the methodology in preparation to test publishing immediately
     ...    ${PUBLICATION_NAME} - Amended methodology
     ...    Edit this amendment
 
+Update the methodology amendment's content
```

Comment:
Don't think you need the comma here. I think `Update the methodology amendment content` reads ok as a test case title

Reply:
I think the original is clearer and less ambiguous, so I'm inclined to leave this. The content belongs to that specific methodology amendment, rather than updating general methodology-amendment content. That said, the intended meaning is easy to understand saying it either way given the context. I don't think this is important in the grand scheme of things.